base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -319,7 +319,7 @@ where T: BlockchainBackend + 'static
                         target: LOG_TARGET,
                         "A peer has requested a block with hash {}", hash_hex
                     );
-                    match self.blockchain_db.fetch_block_with_stxo(hash).await {
+                    match self.blockchain_db.fetch_block_with_transaction_output(hash).await {
```

Comment:
Is not reproducing the same results here, is this intentional? 
STXO != TXO,  TXO = STXO || UTXO